packages/core/src/components/TextField/TextField.tsx

```diff
@@ -177,6 +190,7 @@ TextField.defaultProps = {
     size: 'M',
     type: 'text',
     variant: 'filled',
+    helperAndErrorStateVariant: 'default',
```

Comment:
You can create a `boolean` prop with name `showTooltipForHelperAndErrorText`.

Reply:
I like this idea

Reply:
Done